File: src/com/nerdscorner/mvp/events/utils/FileReader.java
package com.nerdscorner.mvp.events.utils;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;

public class FileReader {

    public static String getFileContents(InputStream inputStream) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
        String line;
        StringBuilder rawFileContent = new StringBuilder();
        while ((line = reader.readLine()) != null) {
            rawFileContent.append(line).append(System.lineSeparator());
        }
        return rawFileContent.toString();
    }
}

File: src/com/nerdscorner/mvp/utils/FileReader.kt
package com.nerdscorner.mvp.utils

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStream
import java.io.InputStreamReader

internal object FileReader {

    @Throws(IOException::class)
    fun getFileContents(inputStream: InputStream): String {
        val reader = BufferedReader(InputStreamReader(inputStream))
        val rawFileContent = StringBuilder()
        for (line in reader.lines()) {
            rawFileContent
                    .append(line)
                    .append(System.lineSeparator())
        }
        return rawFileContent.toString()
    }
}
